bynder_sdk/client/asset_bank_client.py

```diff
@@ -88,7 +89,7 @@ def upload_file(self, file_path: str, brand_id: str,
         """ Upload file.
             Params:
                 file_path: the local filepath of the file to upload.
-                brand_id: the brandid of the brand that belong the asset.
+                brand_id: the brand id of the brand that belong the asset.
```

Comment:
I believe `brand_id` is no longer required.

Reply:
Now that I think of it, the whole `query` hash is no longer used.

Reply:
No I think the `brand_id` is still needed. It's not needed when we update an existing asset (in which case we specify the `mediaId`) which is why we unset it under `save_media` when `mediaId` is present. 

But if it's a new asset upload we need to associate it with a brandId. See [here](https://bynder.docs.apiary.io/#reference/download/download-operations/save-as-a-new-asset) 

There are other optional form fields too like `name` and `description `for instance which will be part of the `query` aka upload_data.